Bond/Bond.swift

```diff
@@ -66,8 +67,8 @@ public class Bond<T> {
   }
   
   public func bind(dynamic: Dynamic<T>, fire: Bool, strongly: Bool) {
-    dynamic.bonds.append(BondBox(self))
-    
+    dynamic.bonds.insert(BondBox(self))
+
     if strongly {
       self.bondedDynamics.append(dynamic)
```

Comment:
We are keeping this as an array? Well, I guess that's OK. We are using it only for retaining and unbinding purposes.